
This module breaks biosdisk, and isn't yet ready to replace it.

--- conf/i386-pc.rmk~	2008-01-06 16:34:11.000000000 +0100
+++ conf/i386-pc.rmk	2008-01-07 12:49:38.000000000 +0100
@@ -136,7 +136,7 @@
 	_multiboot.mod chain.mod multiboot.mod reboot.mod halt.mod	\
 	vbe.mod vbetest.mod vbeinfo.mod video.mod gfxterm.mod \
 	videotest.mod play.mod bitmap.mod tga.mod cpuid.mod serial.mod	\
-	ata.mod vga.mod
+	vga.mod
 
 # For biosdisk.mod.
 biosdisk_mod_SOURCES = disk/i386/pc/biosdisk.c
